framework/bundles/org.jboss.tools.rsp.server.spi/src/main/java/org/jboss/tools/rsp/server/spi/servertype/AbstractServerType.java

```diff
@@ -13,17 +13,17 @@
 import org.jboss.tools.rsp.api.dao.util.CreateServerAttributesUtility;
 
 public abstract class AbstractServerType implements IServerType {
+
 	private String id;
 	private String name;
 	private String desc;
+
 	public AbstractServerType(String id, String name, String desc) {
 		this.id = id;
 		this.name = name;
 		this.desc = desc;
 	}
 
-	public abstract IServerDelegate createServerDelegate(IServer server);
```

Comment:
Why remove?

Reply:
bcs this is already in the interface ```IServerType#createServerDelegate```. There's no need to repeat this in the abstract class.